packages/adobe-activation/config.yml

```diff
@@ -0,0 +1,27 @@
+Id:             'adobe-activation'
+Description:    'Adobe serializer for licensing Adobe apps'
+Version:        '1.0'
+
+Install: null
+
+ChocoPackages: []
+
+Applications: { }
+
+Files: { }
+
+Environment: { }
+
+Services: { }
+
+ScheduledTasks:
+    - 'adobe_activation'
+
```

Comment:
Where are the AdobeSerialization.exe and the other activation files being copied to the disk?

I might have to think about the best way to do this actually (I have an idea), unless you are doing it somewhere else already.